site2/docs/reference-metrics.md

```diff
@@ -217,6 +217,9 @@ All the topic metrics are labelled with the following labels:
 | pulsar_compaction_latency_le_* | Histogram | The compaction latency with given quantile. <br> Available thresholds: <br><ul><li>pulsar_compaction_latency_le_0_5: <= 0.5ms </li><li>pulsar_compaction_latency_le_1: <= 1ms</li><li>pulsar_compaction_latency_le_5: <= 5ms</li><li>pulsar_compaction_latency_le_10: <= 10ms</li><li>pulsar_compaction_latency_le_20: <= 20ms</li><li>pulsar_compaction_latency_le_50: <= 50ms</li><li>pulsar_compaction_latency_le_100: <= 100ms</li><li>pulsar_compaction_latency_le_200: <= 200ms</li><li>pulsar_compaction_latency_le_1000: <= 1s</li><li>pulsar_compaction_latency_le_overflow: > 1s</li></ul> |
 | pulsar_compaction_compacted_entries_count | Gauge | The total number of the compacted entries. |
 | pulsar_compaction_compacted_entries_size |Gauge  | The total size of the compacted entries. |
+| pulsar_txn_tb_active_count | Gauge | Number of active transactions of the topic. |
```

Comment:
```suggestion
| pulsar_txn_tb_active_count | Gauge | The number of active transactions on this topic. |
```
This change applies to others.
Just wonder this metric is "Gauge" while the other two are "Count"? They are all about counting.

Reply:
It cannot be a counter. counter means always increasing, but currently active txns could increase or decrease